pkg/driver/docker/docker.go

```diff
@@ -251,10 +251,20 @@ func (d *dockerDriver) RunDockerDevContainer(
 	}
 
 	// mounts
+	tmpfsMount := true
 	for _, mount := range options.Mounts {
+		// skip tmpfs mount on /tmp if devcontainer specifies another mount option
+		if strings.Contains(mount.String(), "/tmp") {
```

Comment:
maybe need a regex here? So we don't hit `/foo/tmp` cases

Reply:
makes sense yes :+1:  on it